specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2022-10-01-preview/machineLearningServices.json

```diff
@@ -4851,6 +4856,24 @@
       ],
       "type": "object"
     },
+    "ImageMetadata": {
+      "type": "object",
+      "description": "Specifies the Image Metadata about current ComputeInstance",
+      "properties": {
+        "currentImageVersion": {
+          "type": "string",
+          "description": "Indicates the current version of OS Base Image, Compute Instance is running on."
```

Comment:
Specifies the current operating system image version this compute instance is running on.